docs/config/auth0.md

```diff
@@ -114,4 +114,4 @@ function() {
 
 Where `$ADD_USER_METADATA` is the [build output](../../apps/auth0-add-user-metadata/build/index.js) of `@asap-hub/auth0-add-user-metadata`.
 
-**You'll need to force the import of a specific `got` version. On the Auth0 editor, where you find `require('got')` replace by `require('got@11.5.2')`.**
+**You'll need to force the import of a specific `got` version. On the Auth0 editor, where you find `require('got')` replace by `require('got@$VERSION')`, where \$VERSION is the version that `yarn why got` shows for `@asap-hub/auth0-add-user-metadata`.**
```

Comment:
This needs to be a version provided by Auth0. A list can be found at https://auth0-extensions.github.io/canirequire/.

Reply:
ah. I was wondering how this magic actually worked

Reply:
the ideal solution for us would probably be bundling it ourselves, but for now I'll adapt the docs to match

Reply:
I'll actually leave this in place, it's still good to not duplicate the current version information somewhere in markdown, but I will update the READMEs of the packages with a link to those package restrictions